examples/stream/stream.cpp

```diff
@@ -13,7 +13,60 @@
 #include <thread>
 #include <vector>
 #include <fstream>
+#include <ctime>
+
+class SimpleWavWriter {
```

Comment:
- The naming convention is to use snake case, so this should be `class wav_writer`
- Move to `common.h / common.cpp`
- API should be: `open()`, `write()`, `close()`
- Allocate on the stack instead of on the heap with `new` and forgetting to deallocate it
- If `params.save_audio` is true, call `init()`

Reply:
OK